omero/sysadmins/windows/server-installation.txt

```diff
@@ -166,81 +167,30 @@ will need the postgres user's password later in the install.
    this document; other PostgreSQL installation environments are
```

Comment:
Maybe delete the other possibilities here and just leave as "PostgreSQL must be installed as a service".